### PR TITLE
Creating a block with a search string

### PR DESCRIPTION
**Pull request by JuliyaKurbatskaya**

### CODE REVIEW

**Comment by JuliyaKurbatskaya** [comment]
incomplete use of homework.

File: styles/lesson-6.css [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+.trip {
+    width: 205px;
+    height: 40px;
+    margin-right: 684px;
+}
+
+/* текст контейнера trip*/
+span {
+    font-size: 24px;
+    line-height: 28,13px;
```
Comment (Dronom) [inline styles/lesson-6.css]: the style is incorrectly applied and not used
image

File: html/lesson-6.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    <meta name="viewport" content="width=, initial-scale=1.0">
+    <title>Lesson-6</title>
+    <link rel="stylesheet" href="../styles/lesson-6.css">
+</head>
+<body>
+    <section>
+        <div class="big-container">
+            <div class="trip-house">
+                <img class="trip" src="../images/tripHouse.svg" alt="TripHouse">
+                <span>Stay<hr></span>
```
Comment (Dronom) [inline html/lesson-6.html]: Better use :hover instead of <hr>

File: html/lesson-6.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                <span>Attractions</span>
+                <img class="Night" src="../images/Night.svg" alt="Night">
+                <img class="Account" src="../images/AccountCircle.svg" alt="AccountCircle">
```
Comment (Dronom) [inline html/lesson-6.html]: can be made like a list

File: html/lesson-6.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+            <h3>Discover stays <br> to live, work or just relax</h3>
+            <form action="/" method="GET"></form>
+              <div class="form">
+                 <label for="name">Your destination or hotel name</label>
+                  <label for="Check-in">Check-in — Check-out</label>
+                  <label for="quantity"></label><br>
+                  <input id="name" type="text" name="Your destination or hotel name" required placeholder="New York">
+                  <input id="Check-in" type="text" name="Check-in — Check-out" required placeholder="Tue 15 Sept — Sat 19 Sept">
+                  <input id="quantity" type="text"  required placeholder="2 Adults — 0 Children — 1 Room">
+                  <input type="submit" value="Search">
```
Comment (Dronom) [inline html/lesson-6.html]: looks different from the figma template

File: html/lesson-6.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+                <img class="Night" src="../images/Night.svg" alt="Night">
+                <img class="Account" src="../images/AccountCircle.svg" alt="AccountCircle">
+            </div>
+            <div class="relax">
+            <h3>Discover stays <br> to live, work or just relax</h3>
+            <form action="/" method="GET"></form>
+              <div class="form">
+                 <label for="name">Your destination or hotel name</label>
+                  <label for="Check-in">Check-in — Check-out</label>
+                  <label for="quantity"></label><br>
```
Comment (Dronom) [inline html/lesson-6.html]: what is an empty label for? and what is br used for?

File: html/lesson-6.html [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+                  <input id="name" type="text" name="Your destination or hotel name" required placeholder="New York">
+                  <input id="Check-in" type="text" name="Check-in — Check-out" required placeholder="Tue 15 Sept — Sat 19 Sept">
+                  <input id="quantity" type="text"  required placeholder="2 Adults — 0 Children — 1 Room">
+                  <input type="submit" value="Search">
+              </div>  
+            </form>
+            <div class="badge">
+            <img class="google" src="../images/google-play-badge.svg" alt="google-play-badge">
+            <img class="app" src="../images/app-store-badge.svg" alt="app-store-badge">
```
Comment (Dronom) [inline html/lesson-6.html]: it's better to make them links

File: styles/lesson-6.css [hunk 7]
```diff
@@ -29,47 +29,63 @@
...
-section {
+body {
 background-image: url(../images/top-section-back.svg);
 box-sizing: border-box;
 background-size: cover;
 min-height: 100vh;
-overflow-y: hidden;
-
+/*background-repeat: no-repeat;
+background-position: 0 0;*/
```
Comment (Dronom) [inline styles/lesson-6.css:38]: try not to store styles and tags that you don't use

File: styles/lesson-6.css [hunk 8]
```diff
@@ -29,47 +29,63 @@
...
+    list-style: none;
 }
 
 .trip {
     width: 205px;
     height: 40px;
-    margin-right: 684px;
+    /*display: flex;
+    justify-content: flex-start;
+    align-items: flex-start;*/
```
Comment (Dronom) [inline styles/lesson-6.css:75]: the same

**Review by Dronom — COMMENTED** [review]
Looks better

**Pull request merged**